internal/impl/laravel/5.8/handler/generator/route.go

```diff
@@ -0,0 +1,155 @@
+	return routeGenerator.routes
+}
+
+/**
+Add the predefined sets of rest routes for the specific model in the generator routes array
+Parameters:
+	- modelName: name of the model for which routes are to be generated
+	-controller : configuration for controller
+Returns:
+	- instance of the generator object
+Sample Usage:
+	Only that routes will be added thet are in the supported methods array of controller config
+	- AddDefaultRestRoutes('Order')
+	routes generated for the above call are
+	Route::get(/order/get-by-id/{id}, OrderController@get-by-id);
+	Route::get(/order/get-all, OrderController@get-all);
+	Route::post(/order/create, OrderController@create);
+	Route::post(/order/edit/{id}, OrderController@edit);
+	Route::post(/order/delete/{id}, OrderController@delete);
+*/
+func (routeGenerator *RouteGenerator) AddDefaultRestRoutes(modelName string, supportedHttpMethods []string) *RouteGenerator {
+
+	type RouteConfig struct {
+		method         string
+		actionFunction string
+		subURI         string
+		httpMethod     string
+	}
+
+	var apiRouteConfig = []RouteConfig{
+		{actionFunction: "create", method: "POST", subURI: ""},
+		{actionFunction: "edit", method: "PUT", subURI: "{id}"},
+		{actionFunction: "delete", method: "DELETE", subURI: "{id}"},
+		{actionFunction: "getById", method: "GET", subURI: "{id}"},
+		{actionFunction: "getAll", method: "GET", subURI: "all"},
+	}
+
+	for _, routeConfig := range apiRouteConfig {
+
+		/*CHECK IF METHOD IS PRESENT IN SUPPORTEDMETHODS ARRAY OF CONTROLLER IN JSON FILE
+		IF PRESENT ADD THE METHOD ELSE DONT ADD*/
+		if contains(supportedHttpMethods, routeConfig.method) || len(supportedHttpMethods) == 0 {
+			uri := "/" + strings.ToLower(modelName)
+			if routeConfig.subURI != "" {
+				uri = uri + "/" + routeConfig.subURI
+			}
+			action := modelName + "Controller" + "@" + routeConfig.actionFunction
+			routeGenerator.AddRoute(strings.ToLower(routeConfig.method), uri, action)
+		}
+
+	}
+
+	return routeGenerator
+}
+
+/**
+Adds the specific routes to the routes array of generator
+Parameters:
+	- method: http methods like get,post,delete...
+	- uri: uri to perform the action
+	-action: ControllerName@functionName
+Returns:
+	- instance of the generator object
+Sample Usage:
+	- AddRoute("post","/order/create","OrderController@create")
+	Route::post(/order/create, OrderController@create);
+*/
+func (routeGenerator *RouteGenerator) AddRoute(method string, uri string, action string) *RouteGenerator {
+	uri = `"`+uri+`"`
+	action = `"`+action+`"`
+	route := core.NewFunctionCall("Route::" + method)
+	route.AddArg(core.NewParameter(uri)).AddArg(core.NewParameter(action))
+	routeGenerator.routes = append(routeGenerator.Routes(), route)
+	return routeGenerator
+}
+
+/**
+Returns the array of tabbedUnits in which the imports array is followed by routes array
+Returns:
+	- array of tabbed units
+*/
+func (routeGenerator *RouteGenerator) Build() []api.TabbedUnit {
+	var builtRouteFile []api.TabbedUnit
+
```

Comment:
you could do this instead ```buildRoutFile := []api.TabbedUnit{core.NewSimpleStatement(`use Illuminate\Support\Facades\Route`)}``` instead of calling append